kubernetes/samples/scripts/create-rcu-schema/create-rcu-schema.sh

```diff
@@ -65,16 +65,16 @@ fi
 
 echo "ImagePullSecret[$pullsecret] Image[${fmwimage}] dburl[${dburl}] rcuType[${rcuType}]"
 
-dbpod=`kubectl get po | grep oracle | cut -f1 -d " " `
-if [ -z ${dbpod} ]; then
-  echo "Oracle deployment pod not found in [default] namespace"
-  echo "Execute the script start-db-service.sh"
-  exit -2
-fi
+#dbpod=`kubectl get po | grep oracle | cut -f1 -d " " `
```

Comment:
We should not preserve commented out code unless it's a sample.  Why are you retaining this?

Reply:
Removed the commented paragraph

#dbpod=`kubectl get po | grep oracle | cut -f1 -d " " `
#if [ -z ${dbpod} ]; then
#  echo "Oracle deployment pod not found in [default] namespace"
#  echo "Execute the script start-db-service.sh"
#  exit -2
#fi
-
# Make sure the DB deployment Pod is RUNNING
#checkPod ${dbpod} default
#checkPodState ${dbpod} default "1/1"

Reply:
Removed the commented paragraph